torch/jit/__init__.py

```diff
@@ -656,11 +656,16 @@ def script_method(fn, _rcb=None):
 
 def _try_get_weak_module(mod):
     """
-    Get the WeakScriptModuleProxy corresponding to mod if it exists
+    Get the WeakScriptModuleProxy corresponding to mod if it exists or generate
```

Comment:
I am not sure why this changed. It doesn't seem related to the CudnnModule change.